.github/workflows/package.yml

```
@@ -90,17 +90,10 @@ jobs:
     strategy:
       matrix:
-        # Define the runners you want to use
-        runs_on: [ubuntu-24.04, ubuntu-24.04-arm]
-        include:
-          - runs_on: ubuntu-24.04
-            arch_name: x86_64
-          - runs_on: ubuntu-24.04-arm
-            arch_name: aarch64
-    runs-on: ${{ matrix.runs_on }}
+        arch: [x86_64, aarch64]
+    runs-on: ubuntu-24.04${{ case(matrix.arch, 'aarch64', '-arm', '') }}
```

actionlint (1.7.12)

[error] case() requires an odd number of arguments (pred/value pairs + default) but got 4

(expression)